### ISSUE TITLE
Exercises have no base

### ISSUE THREAD

**Issue by deboy69** [wger-project/docker#26]
web_1    | Exercise 2b0205ef-40cc-49d3-b982-d42b60f05ebb has no base, this should not happen!Skipping...
web_1    | Exercise a9f2d62a-6a3c-4b20-b3ff-cb3654bf3195 has no base, this should not happen!Skipping...
web_1    | Exercise 9baf0689-e7d6-4809-9318-01a043860a82 has no base, this should not happen!Skipping...
web_1    | Exercise c81fa1a0-9bfe-474b-b78f-195ef791e499 has no base, this should not happen!Skipping...
web_1    | Exercise ebdd3ac2-dfa2-4c36-8c41-fbcbec16a042 has no base, this should not happen!Skipping...
web_1    | Exercise 9eb10e6a-97e1-46e1-b9ac-226bb25ca080 has no base, this should not happen!Skipping...
web_1    | Exercise ae308b2f-ceb2-4517-93b8-3b90f37fad47 has no base, this should not happen!Skipping...
web_1    | Exercise 87d5edb6-3ecf-4ff3-aa52-38a764e95120 has no base, this should not happen!Skipping...
web_1    | Exercise 2ac4a5f9-2b47-436e-843a-655580746aa3 has no base, this should not happen!Skipping...
web_1    | Exercise 1de6fe0a-f267-4509-84bc-2ecefbe5652d has no base, this should not happen!Skipping...
web_1    | Exercise 6f43fa49-d64e-4132-bebc-2fd722da6997 has no base, this should not happen!Skipping...
web_1    | Exercise d721e53c-afc2-4535-b0ff-61e158dd25f9 has no base, this should not happen!Skipping...
web_1    | Exercise 84794882-9d1d-480b-89ad-4861647b8f82 has no base, this should not happen!Skipping...

Just an FYI

**Comment by deboy69** [comment]
No, did a docker pull/up after changing the compose file to reflect the new changes.

**Comment by marinkiii** [comment]
Same problem here, and I cannot find exercises to add to a workout. also happens after a docker compose down and up. also on the app when searching for a exercise I get an error.
 
error: type '_OneByteString' is not a subtype of the type 'Map<String, dynamic>?'

**Comment by rolandgeider** [comment]
@marinkiii can you delete the postgres volume and spin up everything again?

**Comment by marinkiii** [comment]
> postgres volume and spin up everything again?

Got is working again, but had to delete all volumes and spin up. 

With only deleting the postgres volume it was looking good, but after some changes like downloading fixture data, image and exercise sync, adding new categories and exercises. And doing another docker down and up again same errors. 

But all is working again after deleting all the volumes and images. and doing a docker up -d

Thanks!

**Comment by rolandgeider** [comment]
glad it works now! It's still somewhat strange, I could never reproduce this. In any case the sync logic will be updated soon, it should work a bit better

**Comment by marinkiii** [comment]
Got the same errors again after the latest update, removed all images and volumes and all is working. Still a strange error..

**Comment by rolandgeider** [comment]
This doesn't make any sense. The docker jobs basically do the same thing, why does it fail for some people? This is only logged and printed (neaning there is no exception/traceback) or does this stop the sync?

**Comment by marinkiii** [comment]
well because it is a fresh install, I was creating a new workout and because there are no cardio exercises I create then manually. So I go to the exercise menu item and in the overview, there are no exercises... while a few seconds ago I did an exercise-sync. this is only getting weirder.

But within a workout I can add exercises like hyperextension, but in the workout is shows the number for the exercise but not the text of the exercise.

<img width="658" alt="empty exercise overview" src="https://user-images.githubusercontent.com/113303839/196007736-091f851f-491d-465a-a26f-693b67e7
<img width="529" alt="workout" src="https://user-images.githubusercontent.com/113303839/196007738-340ca191-934d-45b4-a659-d758f345a81c.png">
1cc7.png">

**Comment by rolandgeider** [comment]
That's what I meant, when building the docker Images it always starts as a fresh install.

And why does it still find them on the search box? 😭

**Comment by marinkiii** [comment]
it makes no sense, it only finds the exercises in the workout search box.

**Comment by marinkiii** [comment]
if there is anything I can test or do, just let me know.

**Comment by rolandgeider** [comment]
I guess I could add more logging? But honestly, I don't know what could be done more
